ariadne/cardano/test/Spec.hs

```diff
@@ -168,3 +171,21 @@ expectedAriadneConfig = defaultAriadneCfg
     defaultWalletConfig = acWallet defaultAriadneCfg
     defaultUpdateConfig = acUpdate defaultAriadneCfg
     defaultHistoryConfig = acHistory defaultAriadneCfg
+
+utilSpec :: Spec
```

Comment:
Why is it _util_ spec? :thinking: 
 I think it makes more sense to call it e. g. `cardanoKnitSpec` or something like that :man_shrugging: